week-1/interpret/time-format.js

```diff
@@ -14,16 +14,38 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// There are 7.
 
 // b) How many function calls are there?
+// There isn't any function calls.
 
 // c) Using documentation on MDN, explain what the expression movieLength % 60 represents
 
+// This represents how many seconds are in a minute.
+// This ensures that all the minutes are accounted for, and any remainders will be formatted in seconds.
+
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
 
+// It means that I want you to collect the movieLength which is (8784), I want you to minus the remainingSeconds from this,
+// and because remainingSeconds seconds is stored in a variable, it works out what the sum should be for remainingSeconds,
+// which is (24 seconds). This leaves us with a sum from the expression movieLength - remainingSeconds = (8760),
+// which if then divided by 60 = (146).
+
 // e) What do you think the variable result represents? Can you think of a better name for this variable?
 
+<<<<<<< HEAD
+// The amount of time left of the film
+// const filmTimeLeft = `${remainingHours}:${remainingMinutes}:${remainingSeconds}`;
+
+// f) Try experimenting with different values of movieLength. Will this code work for all values of movieLength? Explain your answer
+
+// The code will work with all values for movieLength. When the result is executed with a value of 1,
+// the timer will display 0(Hours):0(Minutes):1(Seconds).
+// I did notice though that if I changed this number to a ridiculous number for example: 100000000000000 or 1000000000,
```

Comment:
If `movieLength` is `100000000000000` then the formatted time is `9:46:40`.
If `movieLength` is 1000000000 then the formatted time is `1:46:40`
Neither of those values I would say are ridiculous.  The second one is actually a reasonable length of time for a movie.  The first one would be a very long movie.  The code should produce the correct numbers of hours, minutes and seconds for every starting value of `movieTime`